defaults/main.yml

```diff
@@ -1,4 +1,4 @@
-mysql_hardening_enabled: yes
+mysql_hardening_enabled: no
```

Comment:
Up to you, but usually roles are deactivated by default to not accidentally fail builds. For os and ssh that's different, probably because you can successfully execute it on almost build, but mysql have to be executed on mysql-enabled instances only. My use case: I'm using 1 playbook for almost everything, and having group-specific variables that control which roles are enabled (except roles like os/ssh-hardening which can be safelly enabled for all (at least my) instances, so they're enabled by default, though it's an exclusion). We can do the opposite, if you still want. though I think it's better not to leave it disabled and force  user to manually activate it